appveyor.yml

```diff
@@ -7,8 +7,8 @@ install:
 build_script:
   - sbt clean compile
 test_script:
-  - sbt test scripted
+  - sbt test runScripted
```

Comment:
Do you know if it’s possible to have the same matrix thing as we have with travis?

Reply:
Done in the latest version. Take note of the comment about special handling of `^`. It looks related with https://github.com/appveyor/ci/issues/1500.